### DISCUSSION TITLE
Got error: ImportError: no module named 'asyncio'

### DISCUSSION THREAD

**Discussion by PhysicsUofRAUI** [miguelgrinberg/microdot#225] [General]
As the title said I got the error "ImportError: no module named 'asyncio'". I was able to fix it by changing all the time 'asyncio' appeared in the minimal installation file to uasyncio. 

For context I am using the tinyPico board with the Micropython that shipped with it. 

Thought I'd put this in the discussion area because from what I understand Micropython tends to work differently on different devices from different manufacturers so I don't really know if it could be considered a bug. 

I thought I'd share in case anyone else has experienced this.

**Comment by miguelgrinberg** [comment]
What version of micropython are you using? My understanding is that they do not use the 'u' prefix anymore.

**Comment by mattytrentini** [comment]
The internal modules were renamed to drop the 'u' prefix in [MicroPython v1.21](https://github.com/micropython/micropython/releases/tag/v1.21.0) (October 2023). However, importing 'u'module is also allowed to support backwards compatibility (so both `import asyncio` and `import uasyncio` will work).

Versions prior to v1.21 need to import the 'u'module name, so I presume the TinyPICO ships with v1.20 or earlier...

**Comment by PhysicsUofRAUI** [comment]
You all are write. The version that ships with tinypico is 1.19.1 (at least mine).

Thank you for the information!
